CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Changelog
 
-# Unreleased
+# [0.3.0] - 2023-12-20
+
+## **Breaking Changes**
+
+* Pattern API method now require a Unicode `Normalization` strategy in addition to a `CaseMatching` strategy.
 
 ## Bugfixes
 
+* correctly handle Unicode normalization when there are normalizable characters in the pattern
```

Comment:
```suggestion
* correctly handle Unicode normalization when there are normalizable characters in the pattern,
  for example characters with umlauts
```

so it's a little more recognizable to those less versed in unicode

Reply:
good point